Problem: publishing pystarportImage to dockerhub fails

https://github.com/crypto-org-chain/chain-main/runs/3529878782

```
Login Succeeded
28
The push refers to repository [docker.pkg.github.com/crypto-org-chain/chain-main/chain-main-pystarport]
29
f003e077fcff: Preparing
30
fbd7f2bfd527: Preparing
31
0fbb3ddc5d11: Preparing
32
f76dc04ac0a8: Preparing
33
d74b6d2c1bef: Preparing
34
222e3c72e150: Preparing
35
0af2a962775f: Preparing
36
3a7ad550def7: Preparing
37
5ef55d2cdb77: Preparing
38
01e617f32cfa: Preparing
39
8d4a7a80823a: Preparing
40
2f779216c6ab: Preparing
41
a1a6f1ac359e: Preparing
42
ad1d4a4948de: Preparing
43
3864d9c0da60: Preparing
44
d45b09f67e12: Preparing
45
481de926a68e: Preparing
46
1222c37215da: Preparing
47
da6037123313: Preparing
48
8cd219aad327: Preparing
49
bc626a5607e0: Preparing
50
ffdd3d019f28: Preparing
51
56ab80b4ec5a: Preparing
52
ecef3d636bef: Preparing
53
06c1a0d53da4: Preparing
54
a92dc50143a1: Preparing
55
df07f3733f09: Preparing
56
6303d5b9cccd: Preparing
57
97d4b01c8015: Preparing
58
653069216de7: Preparing
59
ac3ca0f97702: Preparing
60
ba4203aa1876: Preparing
61
d184074fd0c7: Preparing
62
f84cf0bd4e97: Preparing
63
534df19c4377: Preparing
64
ee9d815cef77: Preparing
65
ec9ecc7f5c70: Preparing
66
c3a63334f873: Preparing
67
88c9e142f92d: Preparing
68
a151a543984e: Preparing
69
748ef28bb94c: Preparing
70
4b4995b6ae0b: Preparing
71
2f8133f9a98a: Preparing
72
ecb72a5d44fd: Preparing
73
9aeb147f4f00: Preparing
74
0f6f70016fca: Preparing
75
6979e6dbfa1c: Preparing
76
dd41a60e34c8: Preparing
77
ee243c53d34c: Preparing
78
0e2abc370a69: Preparing
79
c6500f93345c: Preparing
80
00e94e397174: Preparing
81
2ccf92d323be: Preparing
82
d59abef3ca44: Preparing
83
42a74c843475: Preparing
84
a4781c48ecb6: Preparing
85
a2d751767686: Preparing
86
7485d45ed6cd: Preparing
87
e1c10e7e400f: Preparing
88
687f18c68001: Preparing
89
86e3c52499cb: Preparing
90
ef84c7a4a5fe: Preparing
91
ff8c58af5f1f: Preparing
92
fa210b20282a: Preparing
93
98a11f18c423: Preparing
94
f4651bbdf39a: Preparing
95
6933fe5ee179: Preparing
96
4931721fcf88: Preparing
97
797c258b3bbe: Preparing
98
f88fe438a206: Preparing
99
076e7ce3343b: Preparing
100
55bc28735881: Preparing
101
501783fa3d8e: Preparing
102
f7ebc8e962e7: Preparing
103
42fb3941acd6: Preparing
104
7675de8bc731: Preparing
105
a08304fddc3b: Preparing
106
a4cfe6000d42: Preparing
107
436034a55e88: Preparing
108
976767b21c5a: Preparing
109
f84cf0bd4e97: Waiting
110
534df19c4377: Waiting
111
ee9d815cef77: Waiting
112
ec9ecc7f5c70: Waiting
113
c3a63334f873: Waiting
114
88c9e142f92d: Waiting
115
a151a543984e: Waiting
116
748ef28bb94c: Waiting
117
4b4995b6ae0b: Waiting
118
2f8133f9a98a: Waiting
119
ecb72a5d44fd: Waiting
120
9aeb147f4f00: Waiting
121
0f6f70016fca: Waiting
122
6979e6dbfa1c: Waiting
123
dd41a60e34c8: Waiting
124
ee243c53d34c: Waiting
125
0e2abc370a69: Waiting
126
c6500f93345c: Waiting
127
00e94e397174: Waiting
128
2ccf92d323be: Waiting
129
d59abef3ca44: Waiting
130
42a74c843475: Waiting
131
a4781c48ecb6: Waiting
132
a2d751767686: Waiting
133
7485d45ed6cd: Waiting
134
e1c10e7e400f: Waiting
135
687f18c68001: Waiting
136
86e3c52499cb: Waiting
137
ef84c7a4a5fe: Waiting
138
ff8c58af5f1f: Waiting
139
fa210b20282a: Waiting
140
98a11f18c423: Waiting
141
f4651bbdf39a: Waiting
142
6933fe5ee179: Waiting
143
4931721fcf88: Waiting
144
797c258b3bbe: Waiting
145
f88fe438a206: Waiting
146
076e7ce3343b: Waiting
147
55bc28735881: Waiting
148
501783fa3d8e: Waiting
149
f7ebc8e962e7: Waiting
150
42fb3941acd6: Waiting
151
7675de8bc731: Waiting
152
a08304fddc3b: Waiting
153
a4cfe6000d42: Waiting
154
436034a55e88: Waiting
155
976767b21c5a: Waiting
156
da6037123313: Waiting
157
8cd219aad327: Waiting
158
bc626a5607e0: Waiting
159
ffdd3d019f28: Waiting
160
56ab80b4ec5a: Waiting
161
ecef3d636bef: Waiting
162
06c1a0d53da4: Waiting
163
a92dc50143a1: Waiting
164
df07f3733f09: Waiting
165
6303d5b9cccd: Waiting
166
97d4b01c8015: Waiting
167
653069216de7: Waiting
168
ac3ca0f97702: Waiting
169
222e3c72e150: Waiting
170
ba4203aa1876: Waiting
171
d184074fd0c7: Waiting
172
0af2a962775f: Waiting
173
3a7ad550def7: Waiting
174
5ef55d2cdb77: Waiting
175
01e617f32cfa: Waiting
176
8d4a7a80823a: Waiting
177
2f779216c6ab: Waiting
178
a1a6f1ac359e: Waiting
179
3864d9c0da60: Waiting
180
d45b09f67e12: Waiting
181
481de926a68e: Waiting
182
1222c37215da: Waiting
183
ad1d4a4948de: Waiting
184
f003e077fcff: Pushed
185
f76dc04ac0a8: Pushed
186
fbd7f2bfd527: Pushed
187
d74b6d2c1bef: Pushed
188
0fbb3ddc5d11: Pushed
189
0af2a962775f: Pushed
190
5ef55d2cdb77: Pushed
191
222e3c72e150: Pushed
192
01e617f32cfa: Pushed
193
3a7ad550def7: Pushed
194
8d4a7a80823a: Pushed
195
a1a6f1ac359e: Pushed
196
2f779216c6ab: Pushed
197
ad1d4a4948de: Pushed
198
3864d9c0da60: Pushed
199
d45b09f67e12: Pushed
200
da6037123313: Pushed
201
1222c37215da: Pushed
202
481de926a68e: Pushed
203
8cd219aad327: Pushed
204
bc626a5607e0: Pushed
205
ffdd3d019f28: Pushed
206
56ab80b4ec5a: Pushed
207
ecef3d636bef: Pushed
208
06c1a0d53da4: Pushed
209
a92dc50143a1: Pushed
210
df07f3733f09: Pushed
211
6303d5b9cccd: Pushed
212
97d4b01c8015: Pushed
213
653069216de7: Pushed
214
ba4203aa1876: Pushed
215
ac3ca0f97702: Pushed
216
d184074fd0c7: Pushed
217
f84cf0bd4e97: Pushed
218
534df19c4377: Pushed
219
ec9ecc7f5c70: Pushed
220
ee9d815cef77: Pushed
221
c3a63334f873: Pushed
222
88c9e142f92d: Pushed
223
a151a543984e: Pushed
224
748ef28bb94c: Pushed
225
4b4995b6ae0b: Pushed
226
2f8133f9a98a: Pushed
227
ecb72a5d44fd: Pushed
228
9aeb147f4f00: Pushed
229
0f6f70016fca: Pushed
230
6979e6dbfa1c: Pushed
231
dd41a60e34c8: Pushed
232
ee243c53d34c: Pushed
233
0e2abc370a69: Pushed
234
c6500f93345c: Pushed
235
00e94e397174: Pushed
236
2ccf92d323be: Pushed
237
d59abef3ca44: Pushed
238
42a74c843475: Pushed
239
a4781c48ecb6: Pushed
240
a2d751767686: Pushed
241
7485d45ed6cd: Pushed
242
e1c10e7e400f: Pushed
243
687f18c68001: Pushed
244
ef84c7a4a5fe: Pushed
245
86e3c52499cb: Pushed
246
ff8c58af5f1f: Pushed
247
fa210b20282a: Pushed
248
6933fe5ee179: Pushed
249
f4651bbdf39a: Pushed
250
4931721fcf88: Pushed
251
797c258b3bbe: Pushed
252
f88fe438a206: Pushed
253
076e7ce3343b: Pushed
254
55bc28735881: Pushed
255
501783fa3d8e: Pushed
256
98a11f18c423: Pushed
257
42fb3941acd6: Pushed
258
f7ebc8e962e7: Pushed
259
7675de8bc731: Pushed
260
a08304fddc3b: Pushed
261
436034a55e88: Pushed
262
976767b21c5a: Pushed
263
a4cfe6000d42: Pushed
264
received unexpected HTTP status: 500 Internal Server Error
265
Error: Process completed with exit code 1.
```

maybe this flow can be removed?

--- a/.github/workflows/nix.yml
+++ b/.github/workflows/nix.yml
@@ -45,22 +45,6 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./coverage.*.txt
           flags: integration_tests
-      - name: Publish docker image
-        if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/')
-        run: |
-          nix-build -A pystarportImage docker.nix
-          IMAGE=$(docker load < result)
-          IMAGE=${IMAGE#Loaded image: }
-          if [[ "$GITHUB_REF" = "refs/tags/"* ]]; then
-            TAG=${GITHUB_REF#refs/tags/}
-          else
-            TAG="latest"
-          fi
-          TARGET="docker.pkg.github.com/${{ github.repository }}/chain-main-pystarport:$TAG"
-          docker tag $IMAGE $TARGET
-          docker login -u "$GITHUB_ACTOR" -p "${{ github.token }}" docker.pkg.github.com
-          docker push $TARGET
-          echo "pushed: $TARGET"
       - name: Tar debug files
         if: failure()
         run: tar cfz debug_files.tar.gz -C /tmp/pytest-of-runner .

